lib/Models/ArcGisFeatureServerCatalogItem.ts

```diff
@@ -410,72 +410,68 @@ export default class ArcGisFeatureServerCatalogItem extends MappableMixin(
     });
   }
 
-  protected forceLoadMapItems(): Promise<void> {
+  protected forceLoadMapItems() {
     const that = this;
-    return that.loadMetadata().then(() => {
-      if (isDefined(that.geoJsonItem)) {
-        return that.geoJsonItem.loadMapItems().then(() => {
-          const featureServerData = that.featureServerData;
-          if (
-            that.useStyleInformationFromService &&
-            featureServerData &&
-            featureServerData.drawingInfo
-          ) {
-            const renderer = featureServerData.drawingInfo.renderer;
-            const rendererType = renderer.type;
-            that.mapItems.forEach(mapItem => {
-              const entities = mapItem.entities;
-              entities.suspendEvents();
-
-              // A 'simple' renderer only applies a single style to all features
-              if (rendererType === "simple") {
-                const simpleRenderer = <SimpleRenderer>renderer;
-                const symbol = simpleRenderer.symbol;
-                if (symbol) {
-                  entities.values.forEach(function(entity) {
-                    updateEntityWithEsriStyle(entity, symbol, that);
-                  });
-                }
-
-                // For a 'uniqueValue' renderer symbology gets applied via feature properties.
-              } else if (renderer.type === "uniqueValue") {
-                const uniqueValueRenderer = <UniqueValueRenderer>renderer;
-                const rendererObj = setupUniqueValueRenderer(
-                  uniqueValueRenderer
-                );
-                entities.values.forEach(function(entity) {
-                  const symbol = getUniqueValueSymbol(
-                    entity,
-                    uniqueValueRenderer,
-                    rendererObj
-                  );
-                  if (symbol) {
-                    updateEntityWithEsriStyle(entity, symbol, that);
-                  }
-                });
-
-                // For a 'classBreaks' renderer symbology gets applied via classes or ranges of data.
-              } else if (renderer.type === "classBreaks") {
-                const classBreaksRenderer = <ClassBreaksRenderer>renderer;
+    if (isDefined(that.geoJsonItem)) {
+      return that.geoJsonItem.loadMapItems().then(() => {
```

Comment:
This is calling a load function in a forceLoad function. Isn't that one of the things this PR tries to remove?

Reply:
Yeah this is a problem.
I should probably re-write this using `GeoJsonMixin`